### ISSUE TITLE
Make version copyable

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/monique-monosynth#37]
I painted the version onto the credit screen last night
Realized I could make it a hyperlink easily, so clicking on it puts full version info on your clipboard. Not that hard and useful as we get into nightly build land
So just wanted to add an issue as a reminder since I won't do it this morning.

**Comment by ghost** [comment]
This either needs a new home or the window needs to change.
Right now this window closes on mouse movement.
No mouse fast enough to get there.

**Comment by baconpaul** [comment]
Yeah I think we may want to update the about screens in both tools

the surge abour screen copyable block has served us super well. I started borrowing it for sc. perhaps we need similar here?
